src/params/types.ts

```diff
@@ -205,10 +205,16 @@ export interface TextInput<T = unknown> {
  */
 export interface ResourceInput {
   resource: {
-    type: string;
+    type: "storage.googleapis.com/Bucket";
   };
 }
 
+export const BUCKET_PICKER: ResourceInput = {
+  resource: {
+    type: "storage.googleapis.com/Bucket",
+  },
+};
+
```

Comment:
Is the helper constant even needed given VS Code will autocomplete the enum string for the resource type?

If so, might prefer naming of `SELECT_STORAGE_BUCKET` over `BUCKET_PICKER`.

Reply:
I think it is much less cringe to type a constant rather than a complex structure that has only one possible configuration. There's precedence here with the ServerValues consts/functions in RTDB/Firestore. Switched to your preferred name though.